jaxlib/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.1.52"
+__version__ = "0.1.53"
```

Comment:
(This change is fine, though.)

Reply:
George recommended waiting for the release to bump the version, I thus reverted this change.